src/utils.ts

```diff
@@ -168,7 +168,7 @@ export function dataConvertDate(
 
 export function toNumber(value: number | Date) {
   if (typeof value === 'undefined') {
-    return 0;
+    return value;
```

Comment:
You will be returning the string "undefined" here

I don't think that is what was intended

Reply:
i want to return undefined if the value is undefined , do you think 0 better? 
or do you have any other suggestion to avoid bugs here?

Reply:
my bad! my brain omitted `typeof` when doing the review and I thought you were doing `value === 'undefined'`

I'm used to doing `value === undefined` when I write code myself